internal/utils/main.go

```diff
@@ -0,0 +1,32 @@
+package utils
+
+import (
+	"encoding/hex"
+	"errors"
+	"time"
+
+	"github.com/stellar/go/xdr"
+)
+
+//PanicOnError is a function that panics if the provided error is not nil
+func PanicOnError(err error) {
+	if err != nil {
+		panic(err)
+	}
+}
+
+//HashToHexString is utility function that converts and xdr.Hash type to a hex string
+func HashToHexString(inputHash xdr.Hash) string {
+	sliceHash := inputHash[:]
+	hexString := hex.EncodeToString(sliceHash)
+	return hexString
+}
+
+//TimePointToUTCTimeStamp takes in an xdr TimePoint and converts it to a time.Time struct in UTC. It returns an error for negative timepoints
+func TimePointToUTCTimeStamp(providedTime xdr.TimePoint) (time.Time, error) {
+	intTime := int64(providedTime)
+	if intTime < 0 {
+		return time.Now(), errors.New("The timepoint is negative")
+	}
+	return time.Unix(intTime, 0).UTC(), nil
```

Comment:
Why does it need the extra `UTC()` function call?

Reply:
Without it, time.Unix defaults to EDT (might just be my timezone). We stored everything in the database in UTC